spec/site/connect_spec.rb

```diff
@@ -51,22 +51,52 @@ def check_sequence version
     end
   end
 
+# Verify that the handshake sequence upon connection is correct for 3.1.1
+#
+# 1. Given the site is new, connected and version 3.1.1
```

Comment:
what is the meaning of "new"?

Reply:
I was thinking of the site object, and couldn't come up with anything better at the time.